sunpy/coordinates/frames.py

```diff
@@ -170,6 +170,35 @@ class HeliographicCarrington(HeliographicStonyhurst):
 
     dateobs = TimeFrameAttributeSunPy()
 
+    def sol_string(self, form='minimal'):
+        datetime = self.dateobs.datetime
+        sol_str_list = [
```

Comment:
Why does it need to be a list? a string could be just concatenated by adding them.
```python
a = 'SOL2017-01-01'
a += 'T00:03:01'
```